src/app/models/Site.ts

```diff
@@ -118,23 +120,22 @@ export class Site extends AbstractModel implements ISite {
       return "";
     }
 
-    return siteMenuItem.route.format({
-      projectId: this.projectIds.values().next().value,
-      siteId: this.id,
-    });
+    return this.getViewUrl(this.projectIds.values().next().value);
```

Comment:
Why is this an observable?

`projectIds` are a primtive array of ints...

Reply:
Its a Set

Reply:
oof that's ugly as sin